### PR TITLE
[RAG-366] Fix Docs Colab links

### PR DESCRIPTION
**Pull request by mendonk**
Moving to the new docs site broke image links.
This PR fixes it.

### CODE REVIEW

**Review by nicoloboschi — APPROVED** [review]

File: docs/modules/ROOT/pages/quickstart.adoc [hunk 1]
```diff
@@ -1,6 +1,11 @@
 = Quickstart
+:navtitle: Quickstart
+:page-layout: tutorial
+:page-icon-role: bg-[var(--ds-neutral-900)]
+:page-toclevels: 1
+:keywords: Machine Learning Frameworks, Embedding Services, Data Warehouses, SDK
```
Comment (jordanrfrazier) [inline docs/modules/ROOT/pages/quickstart.adoc:6]: is this our standard set? Should it have some generative ai words as well?

File: docs/modules/ROOT/pages/quickstart.adoc [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+:page-colab-link: https://colab.research.google.com/github/datastax/ragstack-ai/blob/main/examples/notebooks/quickstart.ipynb
```
Comment (jordanrfrazier) [inline docs/modules/ROOT/pages/quickstart.adoc:7]: I just Viewed the file and don't see a colab link through that - do you also need to add
```
link:{page-colab-link}[Open in Colab]
```

**Review by jordanrfrazier — APPROVED** [review]

File: docs/modules/ROOT/pages/quickstart.adoc (hunk 1 above)
Comment (mendonk) [reply docs/modules/ROOT/pages/quickstart.adoc:6]: It is the standard set

File: docs/modules/ROOT/pages/quickstart.adoc (hunk 2 above)
Comment (mendonk) [reply docs/modules/ROOT/pages/quickstart.adoc:7]: There's now a helper that will render the colab link and icon 
[https://github.com/riptano/docs-ui/blob/main/src/partials/tutorial-info.hbs](tutorial-info.hbs)

**Pull request merged**